modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -191,6 +191,11 @@ type Handler struct {
 	// - `{http.reverse_proxy.header.*}` The headers from the response
 	HandleResponse []caddyhttp.ResponseHandler `json:"handle_response,omitempty"`
 
+	// If set, the proxy will write very detailed logs about its
+	// inner workings. Enable this only when debugging, as it
+	// will produce a lot of output.
+	TraceLogs bool `json:"trace_logs,omitempty"`
```

Comment:
"Trace" isn't the word that correlates in my head, at least, to "verbose debug logs." Can we call this something else? Like "VerboseLogs"?

Also, we should probably document this as experimental and subject to change, as I'm not sure what the best way to configure this is; or the best name for it. But I think `VerboseLogs` could be a good start. What do you think?

Reply:
Sure :+1:

Reply:
Done